sections/recommendations.tex

```diff
@@ -0,0 +1,110 @@
+% !TeX spellcheck = en_US
+\addsection{Recommendations}{\art/haste.png}
+
+\iftoggle{printable}{}{\bigbreak}
+
+\begin{multicols}{2}
+We've gathered up some custom rules and best practices to make your gaming experience as smooth as possible.
+Bear in mind that those are just \textbf{opinions of the authors} of this book.
+Enjoy.
+
+\subsection*{Play Time}
+
+You can estimate your \textbf{minimal} play time using the following formula:
```

Comment:
I like the idea of highlighting the minimum 👍

Reply:
Still thinking about that table 🤔